carbon-design-system/ibm-security
# feat(canary-banner): added banner "Page layouts will remain Canary[...]" inside Page Layouts

## Pull request 
Adding an info banner (InlineNotification) to the top of all Page Layouts.

### Affected issues

- For [Storybook page layout "Canary" banner #257](https://github.ibm.com/security/design-core-experience/issues/257)

### Proposed changes

- Adding a banner (InlineNotification) to the top of all Page Layouts

### Testing instructions

- Go to the UNSTABLE Modular Page Layouts in storybook
- View the banner at the top, which should read "Page layouts will remain Canary until Carbon v11 2x Grid release"
- Click between all Page Layout stories, and the banner should appear at the top of them all
